finance/proposal-calls/cycle3.md

```diff
@@ -0,0 +1,90 @@
+# Cycle III funding requests and ideas
+
+
+# What is different from the previous cycle?
+
+This cycle now includes funding from the NASA ROSES proposal, not just the Moore Grant. Process-wise, most of the below is similar to the Cycle II (at the time called “Phase II”), with some additional context and minor practical changes based on how Phase II progressed. The main difference in this cycle is the “hiring process” mentioned below. Concretely, in this cycle you can now propose a project that you do not necessarily wish to execute yourself, but that you would like Astropy to find someone to do the work. We hope this will serve both to get work done that our current community does not enjoy doing, as well as serving to expand our community. 
```

Comment:
```suggestion
This cycle includes funding from both NASA ROSES and Moore Foundation. Process-wise, most of the below is similar to the Cycle II (at the time called “Phase II”), with some additional context and minor practical changes based on how Phase II progressed. The main difference in this cycle is the “hiring process” mentioned below. Specifically, in this cycle **_you can now request funding for a project that you do not necessarily wish to execute yourself, but that you would like Astropy to find someone to do the work._** We hope this will serve both expand our community and get desired work done that our current community does not have the capacity to execute.  
```